### PR TITLE
Update lucuma-react, scalac

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: explore/src/main/scala/explore/highcharts.scala [hunk 1]
```diff
@@ -28,6 +27,6 @@ def commonOptions: ChartOptions =
   ChartOptions()
     .setStyledMode(true)
     .setAlignTicks(false)
-    .setZooming(ChartZoomingOptions().setType(OptionsTypeValue.xy))
+    .setZooming(ChartZoomingOptions().setType(highchartsStrings.xy.toString))
     .setPanning(ChartPanningOptions().setEnabled(true))
-    .setPanKey(OptionsPanKeyValue.shift)
+    .setPanKey("shift")
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/highcharts.scala:32]: There's no constant anymore for this.

Comment (cquiroz) [reply explore/src/main/scala/explore/highcharts.scala:32]: WDYM, on the facade? Maybe we should have our own

**Review by cquiroz — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/highcharts.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/highcharts.scala:32]: Yup, ST is not generating it anymore.

We can add our own to the facade, yes.

**Review by toddburnside — APPROVED** [review]
LGTM

**Pull request merged**